drivers/dma/xdmac_sam.c

```diff
@@ -0,0 +1,357 @@
+/*
```

Comment:
Rename your driver as dma_xdmac_sam.c

Not sure it's written anywhere but in drivers, we always write names as <domain>_<name>.<c/h>

Reply:
I'll change the name of the driver to dma_sam_xdmac.c to match the other Atmel SAM drivers. That's the second of the two conventions we're using.